tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -643,38 +643,47 @@ export class WebGPUBackend extends KernelBackend {
 
     const size = webgpu_util.GPUBytesPerElement(tensorData.dtype) *
         util.sizeFromShape(tensorData.shape);
-    const buffer =
-        this.bufferManager.acquireBuffer(size, this.defaultGpuBufferUsage());
-
-    tensorData
-        .resourceInfo = {size, usage: this.defaultGpuBufferUsage(), buffer};
+    let buffer;
     if (tensorData.values) {
-      const stagingBuffer = this.bufferManager.acquireUploadBuffer(
-          size, GPUBufferUsage.MAP_WRITE | GPUBufferUsage.COPY_SRC);
-      const arrayBuffer = stagingBuffer.getMappedRange();
-      if (tensorData.dtype === 'int32' || tensorData.dtype === 'bool') {
-        new Int32Array(arrayBuffer).set(tensorData.values as TypedArray);
+      buffer = this.bufferManager.acquireBuffer(
+          size, this.defaultGpuBufferUsage(), true);
```

Comment:
It's hard to say which flags are default usages. Can we remove defaultGpuBufferUsage()?

Reply:
Removing `defaultGpuBufferUsage()` is unrelated changes for this PR. I prefer doing it in a separate PR.

Reply:
OK